app/controllers/declaration/RepresentativeDetailsPageController.scala

```diff
@@ -75,4 +81,13 @@ class RepresentativeDetailsPageController @Inject()(
       case StandardDec =>
         controllers.declaration.routes.CarrierDetailsPageController.displayForm()
     }
+
+  private def updateCache(
+    sessionId: String,
+    formData: RepresentativeDetails
+  ): Future[Either[String, ExportsCacheModel]] =
+    updateHeaderLevelCache(sessionId, model => {
+      val updatedParties = model.parties.map(_.copy(representativeDetails = Some(formData)))
```

Comment:
Can we be sure there will be `Some(parties)`? Otherwise we lose this form data

Reply:
fixed

Reply:
Addressed! :)